src/kirby/components/toolbar/toolbar.component.tns.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  Component,
+  OnInit,
+  Input,
+  ContentChild,
+  TemplateRef,
+  Output,
+  EventEmitter,
+} from '@angular/core';
+import { isAndroid } from 'tns-core-modules/platform';
+
+import { ToolbarEndElementDirective, ToolbarSecondaryElementDirective } from './toolbar.directive';
+
+@Component({
+  selector: 'kirby-toolbar',
+  templateUrl: './toolbar.component.html',
+  styleUrls: ['./toolbar.component.scss'],
+})
+export class ToolbarComponent implements OnInit {
+  @Input() title: string;
+  @Input() showBackButton: boolean;
+  @Input() themeColor: string;
+
+  @Output() backButtonSelected: EventEmitter<null> = new EventEmitter<null>();
+
+  @ContentChild(ToolbarEndElementDirective, { read: TemplateRef }) endElement;
+  @ContentChild(ToolbarSecondaryElementDirective, { read: TemplateRef }) secondaryElement;
+
+  isAndroid(): boolean {
```

Comment:
Should this maybe be a general used `directive` like shown [here](https://docs.nativescript.org/angular/ui/ng-ui-widgets/ng-directives#create-custom-directive)